### ISSUE TITLE
DSN with slave instances not working as expected

### ISSUE THREAD

**Issue by penfold45** [snc/SncRedisBundle#159]
Hi

I have just implemented using a slave redis insatance and I have the following settings

<pre>
snc_redis:
    clients:
        default:
            type: predis
            alias: default
            dsn:
               - redis://%redis.location%/0?alias=master
               - redis://%redis.slave.location%/0
            options:
                replication: true
            logging: %kernel.debug%

        session:
            type: predis
            alias: session
            dsn:
               - redis://%redis.location%/1?alias=master
               - redis://%redis.slave.location%/1
            options:
                replication: true
        profiler:
            type: predis
            alias: profiler
            dsn:
               - redis://%redis.location%/2?alias=master
               - redis://%redis.slave.location%/2
            options:
                replication: true
</pre>

basically the code is now putting all redis keys into database 2 (the last one) and nothing goes into the other.

It does not appear to be directly linked to the alias as the "gets" on the slave are also also looking at database number 2.
